### PR TITLE
Feature/test

### PR DESCRIPTION
**Pull request by PrzerostNadSens**
first version of the tests is the file structure ok?

### CODE REVIEW

File: src/test/spec/recipe.spec.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { chai, expect, app, chaiHttp } from "../test.config";
+chai.use(chaiHttp);
```
Comment (MoodupDev) [inline src/test/spec/recipe.spec.ts:2]: move chai.use(chaiHttp) to test.config file

Comment (MoodupDev) [inline src/test/spec/recipe.spec.ts:2]: move chai.use(chaiHttp) to test.config file

File: src/test/spec/recipe.spec.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+describe("cos", function () {
```
Comment (MoodupDev) [inline src/test/spec/recipe.spec.ts:4]: Declare a more meaningful name or mark this as TODO

File: src/test/spec/user.spec.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+describe("User", function () {
+  describe("POST /user/", function () {
+    it("should create user", async function () {
+      const response = await chai
+        .request(app)
+        .post("/user")
+        .send(createUserPayload);
+      expect(response.error).to.be.false;
+      expect(response).to.have.status(201);
```
Comment (MoodupDev) [inline src/test/spec/user.spec.ts:14]: Check status first, then response.error, then response.body

File: src/test/spec/user.spec.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { chai, expect, app, chaiHttp } from "../test.config";
+chai.use(chaiHttp);
```
Comment (MoodupDev) [inline src/test/spec/user.spec.ts]: Move `chai.use(chaiHttp)` to test.config

File: src/test/test.config.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import dotenv from "dotenv";
+import chai, { expect } from "chai";
+import chaiHttp from "chai-http";
+import faker from "faker";
+chai.use(chaiHttp);
+
+dotenv.config();
+
+process.env.DATABASE_CONNECTION_STRING =
+  "mongodb+srv://Admin:trudne_haslo.123@cluster0.edli7.mongodb.net/recipe-test-db?retryWrites=true&w=majority";
```
Comment (MoodupDev) [inline src/test/test.config.ts:10]: Move secrets to .env

**Review by MoodupDev — CHANGES_REQUESTED** [review]
Check comments

File: src/test/test.config.ts [hunk 6]
```diff
@@ -5,10 +5,10 @@ import faker from "faker";
 chai.use(chaiHttp);
 
 dotenv.config();
+// const DATABASE_NAME = "recipe-test-db";
```
Comment (PrzerostNadSens) [inline src/test/test.config.ts]: comments are deleted in the next commit

File: package.json [hunk 7]
```diff
@@ -48,25 +47,36 @@
...
     "@types/morgan": "^1.9.3",
     "@types/shortid": "0.0.29",
     "@types/swagger-jsdoc": "^6.0.1",
     "@types/swagger-ui-express": "^4.1.3",
     "@types/yamljs": "^0.2.31",
+    "chai": "^4.3.4",
+    "chai-http": "^4.3.0",
     "concurrently": "^6.2.0",
-    "nodemon": "^2.0.12"
+    "dotenv": "^10.0.0",
```
Comment (MoodupDev) [inline package.json:74]: Move `dotenv` to normal dependencies

**Review by MoodupDev — APPROVED** [review]

**Pull request merged**